VideoLocker/res/values/strings.xml

```diff
@@ -168,7 +168,7 @@
 
     <!-- My Courses -->
     <!-- Title of course dashboard screen -->
-    <string name="course_home">Course Home</string>
+    <string name="course_dashboard">Course Dashboard</string>
```

Comment:
I followed the norm in this case (which was sending the activity's name as the screen name), but I did felt odd about it. Fixing it.